scripts/run-dev.mjs

```diff
@@ -0,0 +1,50 @@
+import { spawn } from 'child_process'
+import waitOn from 'wait-on'
+import parcel from './parcel.mjs'
+
+async function waitForTask(taskName) {
+  return new Promise((resolve, reject) => {
+    const ps = spawn('npm', ['run', taskName], { stdio: 'inherit' })
+
+    ps.once('close', (exitCode) =>
+      exitCode === 0 ? resolve() : reject(`${taskName} failed with exit code: ${exitCode}`)
+    )
+
+    ps.once('error', (err) => console.error(`Error executing task ${taskName}`, err))
+  })
+}
+
+async function prepareEnvironment() {
+  const bundle = waitForTask('bundle:bridge')
+  const compile = waitForTask('compile')
+
+  return Promise.all([bundle, compile])
+}
+
+async function launchDevServer() {
+  const { server, host } = await parcel()
```

Comment:
I'd consider renaming the `parcel.mjs` script and this import to something neutral (`bundler.mjs` / `initDevServer`?) in case we use a different bundler in future